src/frontend/views/entity/Crud/index.tsx

```diff
@@ -103,10 +103,16 @@ function useEntityCrudView() {
   }, [entityCrudSettings.data]);
 
   const toggleCrudSettings = (field: keyof IEntityCrudSettings) => {
-    const newState = {
-      ...entityCrudSettingsState,
-      [field]: !entityCrudSettingsState[field],
-    };
+    let newState = {...entityCrudSettingsState};
+
+    if(field.toLocaleLowerCase() === ENTITY_CRUD_SETTINGS_TAB_LABELS.DETAILS.toLowerCase() && entityCrudSettingsState[field]) {
+      newState[ENTITY_CRUD_SETTINGS_TAB_LABELS.UPDATE.toLowerCase()] = !entityCrudSettingsState[field];
+    } else if (field.toLocaleLowerCase() === ENTITY_CRUD_SETTINGS_TAB_LABELS.UPDATE.toLowerCase() && !entityCrudSettingsState[field]) {
+      newState[ENTITY_CRUD_SETTINGS_TAB_LABELS.DETAILS.toLowerCase()] = !entityCrudSettingsState[field];
+    }
+
+    newState[field] = !entityCrudSettingsState[field];
+
```

Comment:
Looks like we can simplify all these to
```ts
    const newState = {
      ...entityCrudSettingsState,
      [field]: !entityCrudSettingsState[field],
    };

    if (field === "details" && !newState.details) {
      newState.update = false;
    } else if (field === "update" && newState.update) {
      newState.details = true;
    }
```
What do you think

Reply:
Got it. Much cleaner